tests/client/core/api/test_api.js

```diff
@@ -187,4 +188,21 @@ describe('api', () => {
         });
     });
   });
+
+  describe('startLoginUrl', () => {
+    it('includes the next path', () => {
+      const location = { pathname: '/foo', query: { bar: 'BAR' } };
+      assert.equal(
+        api.startLoginUrl({ location }),
+        'https://addons.mozilla.org/api/v3/accounts/login/start/?to=%2Ffoo%3Fbar%3DBAR');
```

Comment:
I think there's only one thing in the query string. However there is another test that has `to` and `config` so that could fail. I think ES6 made insertion order stable so this should be fine (for test code, let's not depend on that for implementations).